apps/openmw/mwmechanics/aicombat.cpp

```diff
@@ -490,6 +490,23 @@ namespace MWMechanics
 
     void AiCombatStorage::startCombatMove(bool isDistantCombat, float distToTarget, float rangeAttack, const MWWorld::Ptr& actor, const MWWorld::Ptr& target)
     {
+        // get the range of the target's weapon
+        float rangeAttackOfTarget = 0.f;
```

Comment:
Just a minor nitpick, but you don't need to initialise this only to change its value immediately afterwards.